src/main/resources/spring/migration-bean.xml

```diff
@@ -156,6 +159,10 @@
         <constructor-arg type="java.lang.String" value="index" />
     </bean>
 
+    <bean id="namespaceFile" class="java.io.File">
+        <constructor-arg type="java.lang.String" value="src/test/resources/namespaces.properties"/>
```

Comment:
Moving `migration-bean.xml` to `test/resources` seems reasonable (noting that this may need to change as well: https://github.com/fcrepo4-labs/migration-utils/blob/master/src/main/java/org/fcrepo/migration/Migrator.java#L126).